### PR TITLE
Feat/display detailed report for flood

### PR DESCRIPTION
**Pull request by sifelix**
English
image

Khmer
image

### CODE REVIEW

File: apps/frontend/translations/en.json [hunk 1]
```diff
@@ -221,7 +221,7 @@
       }
     }
   },
-  "provinces": {
```
Comment (ericboucher) [inline apps/frontend/translations/en.json:224]: why the singularization?

File: apps/frontend/components/DisasterTable/DisasterTable.tsx [hunk 2]
```diff
@@ -20,31 +21,41 @@ export const DisasterTable = ({
...
+        '& .MuiDataGrid-cell': {
+          whiteSpace: 'normal !important',
+          wordWrap: 'break-word !important',
+        },
+        '& .MuiDataGrid-columnHeaderTitleContainerContent': {
+          whiteSpace: 'normal !important',
+          wordWrap: 'break-word !important',
+          lineHeight: 'normal',
+        },
+      }}
```
[the remark this answers is not in the document]
Comment (sifelix) [reply apps/frontend/components/DisasterTable/DisasterTable.tsx:42]: Unfortunately no: https://github.com/mui/mui-x/issues/898

File: apps/frontend/components/FormValidation/FloodFormValidation/tablesConfig/NumAffected-1.tsx [hunk 3]
```diff
@@ -4,28 +4,28 @@ import { FLOOD, FloodSpecific } from '@wfp-dmp/interfaces';
...
-  getColumnSetup(FloodSpecific.NumKidMissing, FLOOD),
-  getColumnSetup(FloodSpecific.NumOldMissing, FLOOD),
-  getColumnSetup(FloodSpecific.NumDisMissing, FLOOD),
-  getColumnSetup(FloodSpecific.NumTInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumMeInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumFeInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumKidInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumOldInjure, FLOOD),
-  getColumnSetup(FloodSpecific.NumDisInjure, FLOOD),
+  getColumnSetup(FloodSpecific.NumVillAff, FLOOD, 70),
```
[the remark this answers is not in the document]
Comment (sifelix) [reply apps/frontend/components/FormValidation/FloodFormValidation/tablesConfig/NumAffected-1.tsx:7]: Yes, we will do a big clean up when we will have a better overview of all the tables

File: apps/frontend/translations/en.json (hunk 1 above)
Comment (sifelix) [reply apps/frontend/translations/en.json:224]: it is more logical and I used it in the table header

**Comment by sifelix** [comment]
> @sifelix if there is no data in the column, let's not display it maybe?

It is a good idea.
For now, I just copied the previous behavior. 
It's worth investigating how to do so with the datagrid config

**Pull request merged**